lib/resource/resource.ex

```diff
@@ -94,6 +94,13 @@ defmodule AshPaperTrail.Resource do
         doc: """
         Extensions that should be used by the version resource. For example: `extensions: [AshGraphql.Resource], notifier: [Ash.Notifiers.PubSub]`
         """
+      ],
+      store_inputs?: [
```

Comment:
Now that I think about the security ramifications of this, I'll probably want to expand it to:

```elixir
paper_trail do
  store_inputs? true, except: [:password]
end
```

But that can do in a future PR

Reply:
The scrubbing of sensitive inputs I mentioned above might solve for this specific case.

Reply:
Yeah, I like the scrubbing of sensitive input better

Reply:
made a new issue for it here: https://github.com/ash-project/ash_paper_trail/issues/30
will do a PR off the back of it.